mitodl/ocw-hugo-themes
# fix: non-pdf files show pdf thumbnail

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/1338

#### What's this PR do?

In Studio, all files (txt, c, py, tex, pptx, docx, etc) are `Document`s. To display PDF icon for only PDFs, we need to check the document's file type.

#### How should this be manually tested?
1. Download the course [ht-test-site](https://github.mit.edu/ocw-content-rc/ht-test-site) and place it in your local ocw-content folder. (You might not need to do it manually if you have MIT GitHub and RC content configured)
2. Navigate to your ocw-hugo-theme clone.
3. Run `yarn start ht-site-site`. (This name should match the folder name you have in your content folder)
4. Open [http://localhost:3000/lists/a-resource-list/](http://localhost:3000/lists/a-resource-list/)
5. **Expected Behavior**: (Refer to the screenshots)
    - PDF files have a PDF icon
    - Videos have a video icon
    - All other files have a generic file icon
6. (Optional) If you're interested in comparing this output to what we had previously, you may explore [this page](https://draft-qa.ocw.mit.edu/courses/1-ht-test-site-spring-2023/lists/a-resource-list/).


#### Screenshots (if appropriate)

<img width="310" alt="Screenshot 2023-05-15 at 5 39 40 PM" src="https://github.com/mitodl/ocw-hugo-themes/assets/71316217/9c44a3d5-e5ad-4504-b3c5-886377a2ace6">
<img width="1644" alt="Screenshot 2023-05-15 at 5 39 19 PM" src="https://github.com/mitodl/ocw-hugo-themes/assets/71316217/c5a098c1-516d-48dd-9096-cbf12d57b920">
